akka-docs/src/main/paradox/stream/stages-overview.md

```diff
@@ -1326,10 +1326,28 @@ aggregated to the batched value.
 
 ---------------------------------------------------------------
 
+### extrapolate
+
+Allow for a faster downstream by expanding the last emitted element to an `Iterator`. For example, an
+`Iterator.continually(element)` will cause `extrapolate` to keep repeating the last emitted element. 
+
+All original elements are always emitted unchanged - the `Iterator` is only used whenever downstream signals demand, and
+there are no new elements from the upstream.
+
+Includes an optional `initial` argument to prevent blocking the entire stream when there are multiple producers.
+
+**emits** when downstream stops backpressuring
+
+**backpressures** when downstream backpressures
+
+**completes** when upstream completes
+
+---------------------------------------------------------------
+
 ### expand
 
-Allow for a faster downstream by expanding the last incoming element to an `Iterator`. For example
-`Iterator.continually(element)` to keep repeating the last incoming element.
+Like `extrapolate`, but does not have the `initial` argument, and the `Iterator` is also used in lieu of the original 
+element, allowing for it to be rewritten and/or filtered.
```

Comment:
I think the difference from the new `extrapolate` can be made more clear. Maybe something like "Elements are always passed through the function regardless if there is downstream back pressure or not.". Or maybe link to the longer "understanding extrapolate and expand" section?

Reply:
I think a link is the better idea - the examples contained therein will demonstrate the differences better than any "wordy" elaboration.